Source/Analytics/Concepts/Day.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Dolittle.Concepts;
+
+namespace Concepts
+{
+    /// <summary>
+    /// Represents a specific date as number of days since Unix Epoch.
+    /// </summary>
+    public class Day : ConceptAs<int>
+    {
+        public static implicit operator Day(int value)
+        {
+            return new Day {Value = value};
+        }
+
+        public static Day Of(DateTimeOffset timestamp)
```

Comment:
From? I feel that From isn't an ideal name either, but Of is maybe a bit unclear?

Reply:
I agree that it isn't perfect. But saying `Day.Of(today)` or `Day.Of(yesterday)` makes sense to me. I think using `From` might indicate that there should be a `To` as in an interval?
Another possibility could be `Day.Get(today)`?

Reply:
I agree, this is why i felt from wasn't perfect either. Of is maybe as good an option as Get.

Reply:
I'd go with .From(). As it is a static method, From usually means that you create a data structure of that type from some other structure.

Reply:
OK, we'll do `From` – this seems to be used [other places](https://docs.microsoft.com/en-us/powerquery-m/datetime-from), so hope that will be understandable.